src/_locales/en/messages.json

```diff
@@ -108,10 +108,10 @@
     "message": "Password field names"
   },
   "passff_prefs_url_field_names_label": {
-    "message": "Url field names"
+    "message": "URL field names"
   },
   "passff_prefs_script_howto": {
-    "message": "See passff.py for changing the command environment!"
+    "message": "See passff.py for changing the environment variables of Pass!"
```

Comment:
I'm not sure, whether it's comprehensible to capitalize the name of zx2c4 pass.

Reply:
Good point.
The title of the section `Pass Script` is capitalized since it is an English title I guess.